[Traceback] Expose jit line information

henryhu6 opened the pull request:
Summary: Align unwind::frame with traceback.FrameSummary. Add source line information.

Differential Revision: D53771058




cc @voznesenskym @penguinwu @EikanWang @jgong5 @Guobing-Chen @XiaobingSuper @zhuhaozhe @blzheng @wenzhe-nrv @jiayisunx @peterbell10 @ipiszy @yf225 @chenyang78 @kadeng @muchulee8 @aakhundov @ColinPeppler @amjames @desertfire @chauhang

aaronenyeshi reviewed (commented):
Looks good, but I want to get a second opinion from the original author, Zach.

henryhu6 committed:
[Traceback] Expose jit line information (#119907)

Summary:

Align unwind::frame with traceback.FrameSummary. Add source line information.

Differential Revision: D53771058

henryhu6 commented:
/easycla

henryhu6 commented:
/easycla
